etc/tags.sh

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
+
+set -ex
+
+#
+# This little script tries guessing which executable
+# in the system is exuberant ctags. It first checks whether
+# the environment variable $CTAGS is set, and if so, executes
+# the command given by it, otherwise it first tries locating
+# `exctags`, and then `ctags`.
+#
+# Of course there is a lot of room for sophistication here,
+# but this works for me at the moment.
+#
+
+if [[ x$CTAGS != "x" ]]
+then
+    $CTAGS $@
+    exit
+fi
+
+EXCTAGS=`which exctags`
+if [[ x$EXCTAGS != "x" ]] 
+then
+    $EXCTAGS $@
+    exit
+fi
```

Comment:
This relies on `which` being installed. Idiomatic (and also supported in non-bash, although I see you made this script bash-only) would be to use `type` or `command. E.g. like this:
```sh
#!/bin/sh
set -e

for tags in "$EXCTAGS" exctags ctags-exuberant ctags
do
    command -v "$tags" >/dev/null 2>&1 && exec "$tags" "$@"
done

echo "error, exuberant ctags not found"
exit 1
```
Or, if you prefer to see the exact command that is finally being executed, insert an echo:
```sh
#!/bin/sh
set -e

for tags in "$EXCTAGS" exctags ctags-exuberant ctags
do
    command -v "$tags" >/dev/null 2>&1 || continue
    echo "$tags" "$@"
    exec "$tags" "$@"
done

echo "error, exuberant ctags not found"
exit 1
```